frontend/src/components/organizations/OrganizationInfo.vue

```diff
@@ -216,55 +221,95 @@ export default {
         return {}
       },
     },
-    loading: {
+    uploadedDocuments: {
+      type: Array,
+      default: () => {
+        return []
+      },
+    },
+    loadingAll: {
```

Comment:
Maybe just stick with loading? It's the default for any component/view. If we need separate behaviour (e.g. loadingInclusionPolicy) then those get a different name. All doesn't add much.

Reply:
Fixed.